python/triton/language/extra/hip/libdevice.py

```diff
@@ -0,0 +1,433 @@
+from triton.language import core
+
+
+@core.extern
+def abs(arg0, _builder=None):
+    return core.extern_elementwise(
+        "", "", [arg0], {
+            (core.dtype("int32"), ): ("__ocml_trition_abs", core.dtype("int32")),
+            (core.dtype("int64"), ): ("__ocml_trition_abs", core.dtype("int64")),
+            (core.dtype("fp32"), ): ("__ocml_trition_abs", core.dtype("fp32")),
+            (core.dtype("fp64"), ): ("__ocml_trition_abs", core.dtype("fp64")),
+        }, is_pure=True, _builder=_builder)
+
+
+@core.extern
+def floor(arg0, _builder=None):
+    return core.extern_elementwise(
+        "", "", [arg0], {
+            (core.dtype("fp32"), ): ("__ocml_floor_f32", core.dtype("fp32")),
+            (core.dtype("fp64"), ): ("__ocml_floor_f64", core.dtype("fp64")),
+        }, is_pure=True, _builder=_builder)
+
+
+@core.extern
+def rsqrt(arg0, _builder=None):
+    return core.extern_elementwise(
+        "", "", [arg0], {
+            (core.dtype("fp32"), ): ("__ocml_rsqrt_f32", core.dtype("fp32")),
+            (core.dtype("fp64"), ): ("__ocml_rsqrt_f64", core.dtype("fp64")),
+        }, is_pure=True, _builder=_builder)
+
+
+@core.extern
+def ceil(arg0, _builder=None):
+    return core.extern_elementwise(
+        "", "", [arg0], {
+            (core.dtype("fp64"), ): ("__ocml_ceil_f64", core.dtype("fp64")),
```

Comment:
Nit: Why reordering f64 to be ahead of f32 starting this op? (and then switch back starting exp2?) Would be better to be consistent.

Reply:
> Would be better to be consistent.

Done